content/docs/studio/user-guide/projects-and-experiments/run-experiments.md

```diff
@@ -79,24 +79,47 @@ on `Commit changes`.
 
 ![](https://static.iterative.ai/img/studio/cml_commit.png)
 
-At this point, the new experiment appears in the project's experiment table. If
-you just committed to a new branch, then a new pull request will also have been
-created from the new branch to the base branch.
+## What happens after you submit a new experiment
```

Comment:
Can we use a more descriptive title and maybe add some text and/or list with links to the sections below as a summary? (following existing Studio docs style). Rough idea:

```suggestion
## Follow the experiment results

- [Git commit and PRs]
- [Model training]
- [Live metrics and plots]
```

Reply:
Changed. PTAL.

Reply:
Still seeing the same title?

Reply:
Yes, I didn't change the title as suggested. But I changed the content as per this feedback: `add some text and/or list with links to the sections below as a summary? (following existing Studio docs style).`